src/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -207,8 +207,11 @@ public override Stream GetManifestResourceStream(Type type, string name)
                 throw new ArgumentNullException(nameof(type));
 
             string nameSpace = type?.Namespace;
-            string delimiter = (nameSpace != null && name != null) ? Type.Delimiter.ToString() : null;
-            string resourceName = string.Concat(nameSpace, delimiter, name);
+
+            char c = Type.Delimiter;
```

Comment:
If you feel like it you could change `nameSpace` to `namespace` ...

Reply:
Not without putting an @ in front of it, as it's a reserved keyword.